### ISSUE TITLE
require-description-complete-sentence not working correctly with return tag 

### ISSUE THREAD

**Issue by krutoo** [gajus/eslint-plugin-jsdoc#572]
Hi. Looks like JSDoc plugin parses `return` tag not exactly correctly.

Screenshot from VS Code:
![image](https://user-images.githubusercontent.com/13140130/84142724-c0715a00-aa6e-11ea-954c-f0909e0c99eb.png)

In my ESLint config i have option:
```js
    'jsdoc/require-description-complete-sentence': [
      'error',
      {
        tags: [
          'typedef',
          'property',
        ],
      },
    ],
```

How can i take an expected warning/error?

**Comment by brettz9** [comment]
Please fill out a new issue with all requested fields. I am not clear on what you are getting vs. what you are expecting. Thank you.

**Comment by krutoo** [comment]
@brettz9 I create new issue here: https://github.com/gajus/eslint-plugin-jsdoc/issues/573
